### PR TITLE
Fix building volar extension

### PR DESCRIPTION
**Pull request by wvffle**
- [x] I have read the note above about PRs contributing or fixing extensions
- [ ] I have tried reaching out to the extension maintainers about publishing this extension to Open VSX (if not, please create an issue in the extension's repo using [this template](https://github.com/open-vsx/publish-extensions/blob/HEAD/docs/external_contribution_request.md)).
- [x] This extension has an [OSI-approved OSS license](https://opensource.org/licenses) (we don't accept proprietary extensions in this repository)

## Description

This PR fixes building of [`volar`](https://github.com/johnsoncodehk/volar)

### CODE REVIEW

**Comment by filiptronicek** [comment]
Hi @wvffle, thanks a lot for the PR!

I have changed the extension's ID to align with https://github.com/johnsoncodehk/volar/commit/afd4f4ba3a34b939f158b63ac72729aec65f9cb3 and we're ready to go! After https://github.com/open-vsx/publish-extensions/pull/623 gets merged I think we should implement publishing target-specific releases for this one as well.

**Review by filiptronicek — APPROVED** [review]

**Comment by wvffle** [comment]
> thanks a lot for the PR!

I got frustrated that the extension is so outdated in the repository :p

When can I expect to see the extension updated? :)

> After https://github.com/open-vsx/publish-extensions/pull/623 gets merged I think we should implement publishing target-specific releases for this one as well.

Sure!

**Comment by filiptronicek** [comment]
> When can I expect to see the extension updated? :)

I tried publishing it now, but seems it's having some issues. I'll look into this tomorrow.

File: extensions.json [hunk 1]
```diff
@@ -515,13 +515,13 @@
...
   },
-  "johnsoncodehk.volar": {
+  "Vue.volar": {
     "repository": "https://github.com/johnsoncodehk/volar",
     "location": "extensions/vscode-vue-language-features",
     "custom": [
       "npx pnpm install --no-frozen-lockfile --shamefully-hoist",
       "npx pnpm run build",
-      "cd ./extensions/vscode-vue-language-features && npx vsce package -o extension.vsix"
+      "cd ./extensions/vscode-vue-language-features && npx pnpm run prepack:browser && npx vsce package -o extension.vsix"
```
Comment (johnsoncodehk) [inline extensions.json:524]: `prepack:browser` should be replace by `prepack:node`.

File: extensions.json [hunk 2]
```diff
@@ -515,13 +515,13 @@
...
     ]
   },
   "johnsoncodehk.vscode-typescript-vue-plugin": {
```
Comment (johnsoncodehk) [inline extensions.json:527]: Extension id `johnsoncodehk.vscode-typescript-vue-plugin` is also changed to `Vue.vscode-typescript-vue-plugin`.

**Comment by filiptronicek** [comment]
Hey @johnsoncodehk! Thanks for the review here.

It would be great if you could consider publishing to Open VSX directly from the repository like you do to the Microsoft Marketplace, it would really help a lot. 

If you'd need any help or had questions I will be available to answer them all 🙂 - extensions with more involving build steps like this one can be difficult to maintain in repos with hundreds of extensions.

**Comment by johnsoncodehk** [comment]
@filiptronicek Now Volar extensions are hosted by Vue official, I don't own it, I will discuss this with the Vue team..

Is it possible to notify extension maintainers when publish-extensions fails to publish? So I can try PR in time.

**Comment by filiptronicek** [comment]
Sorry for the delay here, @johnsoncodehk, but here's some thoughts about your comment:

> Now Volar extensions are hosted by Vue official, I don't own it, I will discuss this with the Vue team

Thanks a lot, let me know if you need anything!

> Is it possible to notify extension maintainers when publish-extensions fails to publish? So I can try PR in time.

Sadly this isn't possible. The hope here is that if maintainers care about Open VSX, they will publish the extensions themselves, and implementing notifications for maintainers would go against the "temporary solution" goal of the repository.

**Pull request merged**